IndicoToolkit/IndicoWrapper/IndicoClientWrapper.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using IndicoV2;
+using IndicoV2.Storage.Models;
+using IndicoV2.Jobs.Models;
+
+using IndicoToolkit.Types;
+using IndicoToolkit.Exception;
+
+namespace IndicoToolkit.IndicoWrapper
```

Comment:
It's probably not worth it here, but I hate the name "IndicoWrapper"-- feel same way for the python toolkit...

Reply:
same for "indicoClientWrapper" obviously...

Reply:
Agreed. Wonder what your thoughts are on this (and I don't feel too strongly on this), but I think the idea of having this `indico_wrapper.py` as a general hubspot of functionality within the toolkit can be confusing, given that the toolkit itself is more like an assortment of modules. Could be worthwhile branching out indicowrapper fully, much like what we have for Datasets and Workflow module.

Reply:
I'm totally down-- let's go for it